### PR TITLE
tweak(decrapped/native): remove files not supported

### PR DESCRIPTION
**Pull request by PsychoShock**
### Goal of this PR

Following natives that can't be removed from native decls for compatibility, it will now hide them from the list. This will prevent users to use natives not supported anymore.

### How is this PR achieving the goal

Hide natives not supported anymore from the list.

### This PR applies to the following area(s)

Natives

### Checklist

- [x] Code compiles and has been tested successfully.
- [x] Code explains itself well and/or is documented.
- [x] My commit message explains what the changes do and what they are for.
- [x] No extra compilation warnings are added by these changes.

### Fixes issues

This can close https://github.com/citizenfx/fivem/pull/2504

### CODE REVIEW

**Comment by Mathu-lmn** [comment]
Hi, the feat(native/visibility): hide commit needs a description longer than 8 characters for the PR to be valid. Try to be more descriptive

**Comment by PsychoShock** [comment]
Switching to draft since it might break codegen - will do some tests.

**Review by FabianTerhorst — APPROVED** [review]
Thanks for your contribution. Work fine for me.

**Comment by AvarianKnight** [comment]
This would break code gen, the whole reason these haven't been removed is for ABI stability in C#

File: code/components/citizen-scripting-core/src/ScriptHost.cpp [hunk 1]
```diff
@@ -340,7 +340,7 @@ result_t SAFE_BUFFERS TestScriptHost::InvokeNative(fxNativeContext & context)
 	}
 	else
 	{
-		trace("WARNING: NON-EXISTENT NATIVE %016llx\n", context.nativeIdentifier);
```
Comment (AvarianKnight) [inline code/components/citizen-scripting-core/src/ScriptHost.cpp:343]: Another thing to note is this gets called if *any* native is called incorrectly, saying they're 'not supported' is, imo, less useful than saying the native you tried to call didn't exist

**Pull request closed without merging**